tests/jballerina-unit-test/src/test/java/org/ballerinalang/nativeimpl/jvm/runtime/api/tests/Values.java

```diff
@@ -50,4 +59,40 @@ public static BObject getObject(BString objectName) {
         return ValueCreator.createObjectValue(objectModule, objectName.getValue(), StringUtils.fromString("Waruna"),
                                               14, address);
     }
+
+    public static BArray getParamNames(BObject object, BString methodName) {
+        RemoteMethodType remoteType = getRemoteMethod(object, methodName);
+        if (remoteType == null) {
+            return ValueCreator.createArrayValue(TypeCreator.createArrayType(PredefinedTypes.TYPE_STRING, 0), 0);
+        }
+        FunctionParameter[] parameters = remoteType.getParameters();
+        int len = parameters.length;
+        BString[] params = new BString[len];
+        for (int i = 0; i < len; i++) {
+            params[i] = StringUtils.fromString(parameters[i].name);
+        }
+        return ValueCreator.createArrayValue(params);
```

Comment:
Can't we return an array of a pair(tuple), with the paramName and the defaultability together? Then we wouldn't need two methods? We can have a single function called `getParmeters`?